cloudify_aws/cloudformation/resources/stack.py

```diff
@@ -117,43 +189,32 @@ def create(ctx, iface, resource_config, **_):
     iface.create(params)
 
 
+def test(_value):
```

Comment:
What does this do?

Reply:
It was there before. It handles all the datetime objects returned from the client.